src/commands/createStaticWebApp/IStaticWebAppWizardContext.ts

```diff
@@ -6,14 +6,18 @@
 import { WebSiteManagementClient, WebSiteManagementModels } from '@azure/arm-appservice';
 import { UsersGetAuthenticatedResponseData } from '@octokit/types';
 import { ICreateChildImplContext, IResourceGroupWizardContext } from 'vscode-azureextensionui';
+import { Repository } from '../../git';
 import { BranchData, GitTreeData, OrgForAuthenticatedUserData, RepoData } from '../../gitHubTypings';
+import { DeploymentMethods } from './DeploymentMethodStep';
 
 // creating a dummy repoData/branchData would be an annoying amount of work, so use this type to recognize when users have selected create new repo
 export type CreateNewResource = { name?: string; html_url?: string };
 export interface IStaticWebAppWizardContext extends IResourceGroupWizardContext, ICreateChildImplContext {
     accessToken: string;
     client: WebSiteManagementClient;
 
+    deploymentMethod?: DeploymentMethods;
```

Comment:
`deploymentMethod` feels too similar to "local git" vs. "zip deploy". In fact, I would say the deploymentMethod in both cases is "GitHub"

How about we call this property `createScenario` with `publishToNewRepo` and `connectExistingRepo` as the options

Reply:
Works for me.  We'll have another one called `deployToAzure` or `zipdeploy` or something, but should all make sense still.

Reply:
I ran into a _real_ weird feeling scenario.  If you have workspace that has an existing GitHub repo and do a basic create, you'll pick connect to an existing GitHub repo, but it'll skip all of the org/repo/branch steps and jump immediately to preset builds which made me super confused.

I'm thinking that we should get rid of that auto-detect logic for basic create and just add it to the list of potential CreateScenarios.  Like if the project that you have opened is on GitHub and you are owner to it, put it in the quicklist as "Create static web app with {owner}/{repo}" or maybe use it's .git url?

Reply:
Oh I guess I thought basic create would automatically select `connectExistingRepo` in that case. It would be less "weird" without the prompt, right?

Reply:
I guess that makes sense.  I never really loved the auto-select though.  I always felt like while useful, it was kind of jarring because of the sudden logic jump.  What do you think about adding another createScenario pick in this case?